src/components/Editor/Tabs.css

```diff
@@ -67,13 +67,13 @@
   fill: var(--theme-body-color);
 }
 
-.source-tab .prettyPrint {
-  line-height: 0;
-}
-
-.source-tab .prettyPrint svg {
+.source-tab img.prettyPrint {
+  mask: url("/images/prettyPrint.svg") no-repeat;
+  mask-size: 100%;
```

Comment:
do we need `mask-size` now?

Reply:
For reference because we discussed this through Slack, we need `mask-size` in `Tabs.css` because otherwise the SVG mask will be the size of the SVG itself, so `16x x 16x`.  By using `mask-size`, we limit the size to the `height` and `width` provided.